controllers/ccruntime_controller.go

```diff
@@ -181,6 +222,25 @@ func (r *CcRuntimeReconciler) processCcRuntimeInstallRequest() (ctrl.Result, err
 		}
 	}
 
+	// if ds exists, get all labels
+	err, nodes := r.getNodesWithLabels(map[string]string{"cc-preinstall/done": "true"})
+	if err != nil {
+		r.Log.Info("couldn't GET labelled nodes")
+		return ctrl.Result{}, err
+	}
+
+	if r.ccRuntime.Spec.Config.PreInstall.Image != "" &&
+		len(nodes.Items) < r.ccRuntime.Status.TotalNodesCount &&
+		r.ccRuntime.Status.TotalNodesCount > 0 {
+		r.Log.Info("inside != nil check")
```

Comment:
Is the above log statement needed? or can it be reworded ?

Reply:
no, it's a leftover, I'll remove it and look through the other log statements I added to clean everything up

Reply:
this one is also removed now